packages/kit/src/types/global/index.ts

```diff
@@ -0,0 +1,13 @@
+import { ViteImportMeta } from './vite'
+import { WebpackImportMeta } from './webpack'
+
+export type BundlerImportMeta = ViteImportMeta & WebpackImportMeta
```

Comment:
Enforce using either ?

```suggestion
export type BundlerImportMeta = ViteImportMeta | WebpackImportMeta
```